### PR TITLE
updated add_ems_emissions_reporting measure:

### PR DESCRIPTION
**Pull request by rawadelkontar**
updated add_ems_emissions_reporting measure:
- created choice arguments for all arg inputs
- registered errors if argument option is not valid
- changes kg unit to mt
- reported emissions intensity (kg/ft2)

### CODE REVIEW

File: lib/measures/add_ems_emissions_reporting/measure.rb [hunk 1]
```diff
@@ -193,11 +332,23 @@ def run(model, runner, usr_args)
     his_yr_data.each {|r| ems_prgm.addLine("SET his_yr_val = #{r[annual_historical_subregion]}") if r[0].to_i == annual_historical_year}
     ems_prgm.addLine('SET elec = Ele_Sen')
     ems_prgm.addLine('SET conv = 1000000 * 60 * 60') # J to MWh (1000000J/MJ * 60hr/min * 60 min/sec)
-    ems_prgm.addLine('SET fut_hr = fut_hr_val * elec / conv')
-    ems_prgm.addLine('SET his_hr = his_hr_val * elec / conv')
-    ems_prgm.addLine('SET fut_yr = fut_yr_val * elec / conv')
-    ems_prgm.addLine('SET his_yr = his_yr_val * elec / conv')
-
+    ems_prgm.addLine('SET conv_kg_mt = 0.001') # kg to metric ton
```
Comment (bonnema) [inline lib/measures/add_ems_emissions_reporting/measure.rb:335]: @nllong is metric tons okay for what you're doing

**Review by bonnema — CHANGES_REQUESTED** [review]
I think the only required change is the conversion factor.

File: lib/measures/add_ems_emissions_reporting/measure.rb (hunk 1 above)
Comment (nllong) [reply lib/measures/add_ems_emissions_reporting/measure.rb:335]: yes, we need to change to metric tons!

Comment (bonnema) [reply lib/measures/add_ems_emissions_reporting/measure.rb:335]: oh, even better! :grin:

File: lib/measures/add_ems_emissions_reporting/measure.rb [hunk 2]
```diff
@@ -193,11 +332,23 @@ def run(model, runner, usr_args)
...
+    ems_prgm.addLine('SET fut_hr = (fut_hr_val * elec / conv) * conv_kg_mt')
+    ems_prgm.addLine('SET his_hr = (his_hr_val * elec / conv) * conv_kg_mt')
+    ems_prgm.addLine('SET fut_yr = (fut_yr_val * elec / conv) * conv_kg_mt')
+    ems_prgm.addLine('SET his_yr = (his_yr_val * elec / conv) * conv_kg_mt')
+
+
+    #### add emissions intensity metric
+    # get building from model 
+    building = model.getBuilding
+    floor_area = building.floorArea * 3.28084 #change from m2 to ft2
```
[the remark this answers is not in the document]
Comment (rawadelkontar) [reply lib/measures/add_ems_emissions_reporting/measure.rb:345]: Yes, Thanks for catching this !

**Review by bonnema — APPROVED** [review]
looks good!

**Review by DavidGoldwasser — APPROVED** [review]

**Pull request merged**